sympy/solvers/solveset.py

```diff
@@ -871,3 +872,162 @@ def solveset(f, symbol=None):
             return solveset_real(f, symbol)
         else:
             return solveset_complex(f, symbol)
+
+
+###############################################################################
+################################ LINSOLVE #####################################
+###############################################################################
+
+
+def eq_to_matrix(equations, *symbols):
```

Comment:
This is a very generic name. Wouldn't it be more clear if you call it `linear_eq_to_matrix`?